go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	go.opentelemetry.io/otel v1.24.0 // indirect
 	go.opentelemetry.io/otel/metric v1.24.0 // indirect
 	go.opentelemetry.io/otel/trace v1.24.0 // indirect
-	golang.org/x/crypto v0.25.0 // indirect
+	golang.org/x/crypto v0.25.0
```

Comment:
I don't see how this change could have caused this. Did you run `go mod tidy`?

Reply:
Yes, I did.